code/modules/admin/verbs/adminhelp.dm

```diff
@@ -487,6 +487,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
 		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
 		return
+	to_chat(usr, config.policies["adminhelpnotification"])

```

Comment:
they get told about the notification, after they've already sent their ticket, and they can't actually stop it?

Reply:
eee let me fix that